Got some warnings when building with gcc 4.8

../tests/rebase/merge.c:514:14: warning: unused parameter 'path' [-Wunused-parameter]
  const char *path,
              ^
../tests/rebase/merge.c:515:9: warning: unused parameter 'completed_steps' [-Wunused-parameter]
  size_t completed_steps,
         ^
../tests/rebase/merge.c:516:9: warning: unused parameter 'total_steps' [-Wunused-parameter]
  size_t total_steps,
         ^

--- a/tests/rebase/merge.c
+++ b/tests/rebase/merge.c
@@ -517,6 +517,11 @@ void rebase_checkout_progress_cb(
 	void *payload)
 {
 	int *called = payload;
+
+	GIT_UNUSED(path);
+	GIT_UNUSED(completed_steps);
+	GIT_UNUSED(total_steps);
+
 	*called = 1;
 }
 

